tests/functional/adapter/seed_column_types/fixtures.py

```diff
@@ -0,0 +1,111 @@
+_MACRO_TEST_IS_TYPE_SQL = """
+{% macro simple_type_check_column(column, check) %}
```

Comment:
Minor nit, but whenever I see an if block with a bunch of conditions like this, I prefer to use a dictionary. I think this can be done in jinja? It would look something like this:
```
_MACRO_TEST_IS_TYPE_SQL = """
{% macro simple_type_check_column(column, check) %}
    {% set checks = {
        'string': column.is_string,
        'float': column.is_float,
        'number': column.is_number,
        'numeric': column.is_numeric,
        'integer': column.is_integer,
    } %}
    {% if check in checks %}
        {{ return(checks[check]()) }}
    {% else %}
        {% do exceptions.raise_compiler_error('invalid type check value: ' ~ check) %}
    {% endif %}
{% endmacro %}
```